config/initializers/assets.rb

```diff
@@ -5,9 +5,3 @@
 
 # Add additional assets to the asset load path.
 # Rails.application.config.assets.paths << Emoji.images_path
-Rails.application.config.assets.paths << Rails.root.join("app", "assets", "animations")
```

Comment:
I think we can remove the files in app/asserts/animations in a next pr. It was added here: https://github.com/ruby-ui/web/commit/1a2f2635764be56dabed9c237803d8ac63082e0b But I don't think it is being used.